src/app.jsx

```diff
@@ -112,10 +112,9 @@ class AppActive extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            notifications: {},
+            notifications: [],
```

Comment:
The commit message is a bit confusing: "There is no reason for notifications list to be an array". This commit changes it from a map to a list, and "list" and "array" are pretty much the same.

Reply:
Nice catch, I meant to say There is no reason for notifications list to be an object